README.md

```diff
@@ -219,7 +231,11 @@ the default disabled, test it, and move the default from istio-system to istio-c
 
 
 ```bash
-    iop istio-control istio-autoinject $IBASE/istio-control/istio-autoinject --set enableNamespacesByDefault=true
+    # determine whether the istio-cni is installed
```

Comment:
I would avoid this - the recommendation is for user to have a user-values.yaml where he specifies various settings.

For upgrade and gradual adoption we want CNI to work nicely with injected pods with iptables.

Reply:
Do you think I should explain the enable of CNI more in words rather than a jsonpath example?

It seems like the point of the README is to explicitly show some basic `iop` usage for the specific features being installed.  So, I was just showing how they could detect the istio-cni and set flag to enabled/disable the init container.